pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -77,19 +78,25 @@ const (
 
 // GCECloud is an implementation of Interface, LoadBalancer and Instances for Google Compute Engine.
 type GCECloud struct {
+	ClusterID ClusterID
```

Comment:
I'll add a comment to this affect - other controllers, such as ingress, can utilize this instead of having their own logic.